src/utils/strings.ts

```diff
@@ -22,9 +22,22 @@ export const removeUnderscoresAndCapitalize = (string: string): string => {
         .join(' ');
 };
 
-/** Unix timestamp to human readable string */
-export const unixTimestampToHumanReadable = (timestamp: number | null): string => {
-    return timestamp ? new Date(timestamp * 1000).toLocaleString() : 'N/A';
+/** Epoch unix timestamp in seconds to ISO 8601 */
+export const epochTimestampToIso = (
+    timestamp: string | number | null | undefined,
+    inMilliseconds?: boolean
+): string => {
+    if (timestamp === null || timestamp === undefined) {
```

Comment:
nitpick: I have seen a empty util at some point

Reply:
probably not in this project, but good idea

Reply:
You mean i have seen that kind of utils once in my life, oh my :)